docs/excel/custom-functions-overview.md

```diff
@@ -31,16 +31,16 @@ If you use the [Yo Office generator](https://github.com/OfficeDev/generator-offi
 
 | File | File format | Description |
 |------|-------------|-------------|
-| **./src/functions/functions.js**<br/>or<br/>**./src/functions/functions.ts** | JavaScript<br/>or<br/>TypeScript | Contains the code that defines custom functions. |
-| **./src/functions/functions.json** | JSON | Contains metadata that describes custom functions and enables Excel to register the custom functions and make them available to end users. |
-| **./src/functions/functions.html** | HTML | Provides a &lt;script&gt; reference to the JavaScript file that defines custom functions. |
+| **./src/functions.js**<br/>or<br/>**./src/functions.ts** | JavaScript<br/>or<br/>TypeScript | Contains the code that defines custom functions. |
```

Comment:
files *names* are incorrect (should be **customfunctions.js** and **customfunctions.ts**)